docs/src/content/docs/zh-cn/guides/authoring-content.md

```diff
@@ -205,6 +221,149 @@ var fun = function lang(l) {
+
+Expressive Code 提供了几种自定义你的代码示例视觉外观的选项。
+其中许多可以组合使用，以获得极具说明性的代码示例。
+请探索 [Expressive Code 文档](https://github.com/expressive-code/expressive-code/blob/main/packages/%40expressive-code/plugin-text-markers/README.md) 以了解可用的众多设置项。
+下面显示了一些最常见的示例：
+
+- [使用 `{ }` 标记标出整行和行范围](https://github.com/expressive-code/expressive-code/blob/main/packages/%40expressive-code/plugin-text-markers/README.md#marking-entire-lines--line-ranges):
+
+  ```js {2-3}
+  function demo() {
+    // 这一行 (#2) 以及下一行被高亮显示
+    return '这是本代码段的第 3 行';
+  }
+  ```
+
+  ````md
+  ```js {2-3}
+  function demo() {
+    // 这一行 (#2) 以及下一行被高亮显示
+    return '这是本代码段的第 3 行';
+  }
+  ```
+  ````
+
+- [使用 `" "` 标记或正则表达式标出文本字符串](https://github.com/expressive-code/expressive-code/blob/main/packages/%40expressive-code/plugin-text-markers/README.md#marking-individual-text-inside-lines):
+
+  ```js "单个词组" /甚.*表达式/
+  // 单个词组也能被高亮显示
+  function demo() {
+    return '甚至支持使用正则表达式';
+  }
+  ```
+
+  ````md
+  ```js "单个词组" /甚.*表达式/
+  // 单个词组也能被高亮显示
+  function demo() {
+    return '甚至支持使用正则表达式';
+  }
+  ```
+  ````
+
+- [使用 `ins` 或 `del` 来标记行或文本为插入或删除](https://github.com/expressive-code/expressive-code/blob/main/packages/%40expressive-code/plugin-text-markers/README.md#selecting-marker-types-mark-ins-del):
+
+  ```js "return true;" ins="插入" del="删除"
+  function demo() {
+    console.log('这是插入以及删除类型的标记');
+    // 返回语句使用默认标记类型
+    return true;
+  }
+  ```
+
+  ````md
+  ```js "return true;" ins="插入" del="删除"
+  function demo() {
+    console.log('这是插入以及删除类型的标记');
+    // 返回语句使用默认标记类型
+    return true;
+  }
+  ```
+  ````
+
+- [混合语法高亮和类 `diff` 语法](https://github.com/expressive-code/expressive-code/blob/main/packages/%40expressive-code/plugin-text-markers/README.md#combining-syntax-highlighting-with-diff-like-syntax):
+
+  ```diff lang="js"
+    function thisIsJavaScript() {
+      // 这整个代码块都会被作为 JavaScript 高亮
+      // 而且我们还可以给它添加 diff 标记！
+  -   console.log('旧的不去')
+  +   console.log('新的不来')
+    }
+  ```
+
+  ````md
+  ```diff lang="js"
+    function thisIsJavaScript() {
+      // 这整个代码块都会被作为 JavaScript 高亮
+      // 而且我们还可以给它添加 diff 标记！
+  -   console.log('旧的不去')
+  +   console.log('新的不来')
+    }
+  ```
+  ````
+
+#### 边框和标题
+
+代码块可以在类似窗口的框架中呈现。
+默认情况下 shell 脚本语言（例如 `bash` 或 `sh`）会使用一个看起来像终端窗口的边框。
+其他语言在提供了标题的情况下会在一个看起来像代码编辑器的边框中显示。
+
+一个代码块的可选标题可以通过在代码块的开头反引号后面添加一个 `title="..."` 属性来设置，或者通过在代码的前几行中添加一个文件名注释来设置。
+
+- [通过注释添加一个文件名标签](https://github.com/expressive-code/expressive-code/blob/main/packages/%40expressive-code/plugin-frames/README.md#adding-titles-open-file-tab-or-terminal-window-title)
+
+  ```js
+  // my-test-file.js
+  console.log('Hello World!');
+  ```
+
+  ````md
+  ```js
+  // my-test-file.js
+  console.log('Hello World!');
+  ```
+  ````
+
+- [给终端窗口添加一个标题](https://github.com/expressive-code/expressive-code/blob/main/packages/%40expressive-code/plugin-frames/README.md#adding-titles-open-file-tab-or-terminal-window-title)
+
+  ```bash title="Installing dependencies…"
```

Comment:
Would it make sense to translate the frame title too?

Reply:
I intentionally left this title untranslated because I thought this was taken from the output of `npm install`, and because `npm` doesn't have localized terminal messages I kept this title in English too.
But turns out `npm install` doesn't print `Installing dependencies…` so yes this should be translated.